### PR TITLE
Update videos.

### PR DESCRIPTION
**Pull request by kosmikus**
This is resulting from the discussion at #288 and implements the compromise suggested by @tomjaguarpaw in that thread.

### CODE REVIEW

**Review by tomjaguarpaw — APPROVED** [review]
Thanks! We'll need to wait for one more reviewer before merge.

**Comment by Tritlo** [comment]
Looks good to me! I like the selection also, much more familiar than the previous one.

**Pull request merged**
